src/bearer.rs

```diff
@@ -109,4 +110,21 @@ mod tests {
         assert_eq!(None, bearer.refresh_token);
         assert_eq!(None, bearer.expires);
     }
+
+    #[test]
+    fn round_trip() {
```

Comment:
I do not plan to merge the RP, but to mention: this tests would work even if `Bearer` had no fields.